email_template_config/models/mail_template.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    force_email_send = fields.Boolean(string="Force mail send?")
+
+    def send_mail(
+        self,
+        res_id,
+        force_send=False,
+        raise_exception=False,
+        email_values=None,
+        notif_layout=False,
```

Comment:
[in 16.0](https://github.com/OCA/OCB/blob/16.0/addons/mail/models/mail_template.py#L314), this argument has been renamed to `email_layout_xmlid`.

Reply:
fixed